.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.19"
+          go-version: "1.20"
```

Comment:
Sadly, I needed to upgrade to go 1.20 due to a requirement in IAVL v1 on a package called "unsafe"